### PR TITLE
Add a golden crayon for the head of personnel to have.

### PR DESCRIPTION
**Pull request by stonepillars**
## About the PR 
Add a golden crayon, which is distinct from a gold-plated crayon, as this one can write in gold material forever<sup>\*</sup>. It can be found in the Head of Personnel's office on all rotation maps, other than Manta. (EDIT: or Clarion)

<sup>\* Technically, 123,456 times. Good as forever!</sup>

## Why's this needed? 

The Head of Personnel lacks a consistent theft objective item. The only one they have is the First Dollar on Manta. The golden crayon has been added for the other maps. It's also a smaller but equally awesome counterpart to the Captain's golden toilet.

## Changelog

```changelog
(u)Stonepillar
(*)Added the HoP's golden crayon, which can be found in their office on most maps. Beware of possible theft!
```

### CODE REVIEW

**Review by flrsh — APPROVED** [review]

**Comment by Carbadox** [comment]
How about making it a possible job reward?

**Comment by stonepillars** [comment]
> 
> 
> How about making it a possible job reward?

This is more along the lines of the Captain's golden toilet, or the HoS's war medal, its something you can use (You can take a shit on the golden toilet, or wear the HoS medal). This is no different, you can use it to streak the floors in golden crayon wax.

Something more appropriate for a HoP job reward would be a special ID card, a fancy stamp, maybe a cooler chair? This is something meant to be there every round (unless something similar exists, see Manta's First Bill)

File: maps/clarion.dmm [hunk 1]
```diff
@@ -27664,15 +27664,16 @@
 "aYe" = (
 /obj/machinery/light_switch/north,
 /obj/table/wood/auto,
+/obj/machinery/firealarm{
```
Comment (ZeWaka) [inline maps/clarion.dmm:27667]: no changes are permitted to clarion at this time

if you wish to make changes you must ask dions

**Review by ZeWaka — CHANGES_REQUESTED** [review]
^

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**